drivers/bridge/setup_ip_tables.go

```diff
@@ -10,21 +10,22 @@ import (
 
 // DockerChain: DOCKER iptable chain name
 const (
-	DockerChain    = "DOCKER"
-	IsolationChain = "DOCKER-ISOLATION"
+	DockerChain     = "DOCKER"
+	IsolationChain  = "DOCKER-ISOLATION"
+	IsolationChain2 = "DOCKER-ISOLATION-REV"
```

Comment:
I thought `REV` for `reverse` as it goes had in hand with `DOCKER-ISOLATION` chain.
One filter by source, the other one by destination.
I chose a short suffix because the chain name is already long.

Would expand it to `DOCKER-ISOLATION-REVERSE` make it better ?

Or should I change also the existing one and have two different suffixes, like `FORWARD` and `REVERSE` or `SRC` and `DST` ?

In the meantime I will replace the variable names to something more descriptive. Or I will add a comment.